exercises/concept/resistor-color/.docs/instructions.md

```diff
@@ -31,7 +31,8 @@ The goal of this exercise is to create a way:
 - to convert the numerical value into a string representing color
 - to list the different band colors
 
-For tasks number two and three, you will need external crates [`enum-iterator`](https://docs.rs/enum-iterator/1.1.1/enum_iterator/) and [`int-enum`](https://docs.rs/int-enum/0.4.0/int_enum/), which are included in this exercise's `Cargo.toml`. Be sure to check the crates' documentation to learn how to use them.
+For task number one, you will need the external crate [`enum-iterator`](https://docs.rs/enum-iterator/1.1.1/enum_iterator/) and for task number three, you'll need [`int-enum`](https://docs.rs/int-enum/0.4.0/int_enum/). They're both already included in this exercise's `Cargo.toml`. Be sure to check the crates' documentation to learn how to use them.
```

Comment:
Are you sure? Isn't it the other way around? `enum-iterator` helps with listing the colours, which is the third task, and `int-enum` provides `int_value` which converts a colour to a value, which is the first task.

Reply:
You're totally right, nice catch